### ISSUE TITLE
Distribution Support Request

### ISSUE THREAD

**Issue by DoctorWho8** [PowerShell/PowerShell#24195]
### Name of the Distribution

Slackware

### Version of the Distribution

15.0

### Package Types

- [ ] Deb
- [ ] RPM
- [X] Tar.gz
- [ ] Snap - Stop! Please file your issue in [PowerShell-Snap](https://github.com/powershell/powershell-snap) instead.

### Processor Architecture

X86-X64

### .NET Core Support

- [X] The version and architecture of the Distribution is [supported by .NET Core](https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0-supported-os.md#linux).

### Distribution Requirements

- [X] The version of the Distribution is supported for at least one year.
- [ ] The version of the Distribution is not an [interim release](https://ubuntu.com/about/release-cycle) or equivalent.

### Progress

- [ ] An issue has been filed to create a Docker image in [PowerShell-Docker](https://github.com/powershell/powershell-docker)

### For PowerShell Team **ONLY**

- [ ] Docker image created
- [ ] Docker image published
- [ ] Distribution tested
- [ ] Update `packages.microsoft.com` deployment
- [ ] [Lifecycle](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/PowerShell-Support-Lifecycle.md) updated
- [ ] Documentation Updated

**Comment by DoctorWho8** [comment]
Slackware predates all of the shown distributions by many years.  There are many contributors, and I can easily use someone else's packaging tools to create a package for the current release of Powershell for Slackware. But for now building there is not supported. It should be because it is marginally easier.

**Comment by 237dmitry** [comment]
There is [slackbuild](https://slackbuilds.org/repository/15.0/system/powershell/?search=powershell) (precompiled)

**Comment by DoctorWho8** [comment]
> There is [slackbuild](https://slackbuilds.org/repository/15.0/system/powershell/?search=powershell) (precompiled)

That's a build from a compressed tar release of PowerShell. I'm bringing up the problem that someone, myself, possibly can't build from source code because the tools included don't recognize Slackware. At least they did not back in the 14.0 series which was about six years ago. Now I'll go ahead and try again and capture the output and present it here.

**Comment by DoctorWho8** [comment]
Follow up:
I just followed the good steps and it responded:
Get-PowerShell MASTER Installer Version 1.2.0
Installs PowerShell and Optional The Development Environment
  Original script is at: https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools\install-powershell.psh
Arguments used:

Operating System Details:
  OS: linux
  DIST:
  DistroBasedOn:
  PSUEDONAME:
  REV:
  KERNEL: 6.6.36.3-microsoft-standard-WSL2
  MACH: x86_64
  OSSTR:
Sorry, your operating system is based on  and is not supported by PowerShell or this installer at this time.

I'll see if it is possible to invoke the PowerShell script I saw there.

**Comment by 237dmitry** [comment]
> That's a build from a compressed tar release of PowerShell

Yes, I always use it myself (Arch). There is no need to build anything, just unpack, chmod pwsh, symlink and use.

**Comment by rhubarb-geek-nz** [comment]
@DoctorWho8 I notice you ticked the box marked "The version and architecture of the Distribution is [supported by .NET Core](https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0-supported-os.md#linux)." when Slackware is absent from that list.

I suggest that before there can be a PowerShell for Slackware there needs to be a dotnet. sdk

**Comment by theJasonHelmick** [comment]
Thank you for this request. We have no plans to support Slackware.
